File: src/Calculator/Common/ViewState.cpp
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

#include "pch.h"
#include "ViewState.h"

namespace CalculatorApp
{
    namespace ViewState
    {
        Platform::StringReference Snap(L"Snap");
        Platform::StringReference DockedView(L"DockedView");

        bool IsValidViewState(Platform::String ^ viewState)
        {
            return viewState->Equals(ViewState::Snap) || viewState->Equals(ViewState::DockedView);
        }
    }
}

File: src/Calculator/Common/ViewState.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace CalculatorApp
{
    static class ViewState
    {
        public static readonly string Snap = "Snap";
        public static readonly string DockedView = "DockedView";

        public static bool IsValidViewState(string viewState)
        {
            return (viewState == Snap) || (viewState == DockedView);
        }
    }
}

